core/src/platform.h

```diff
@@ -33,3 +38,6 @@ std::string stringFromResource(const char* _path);
  * allocated file
  */ 
 unsigned char* bytesFromResource(const char* _path, unsigned int* _size);
+
+bool fetchData(std::unique_ptr<std::string> _url, std::stringstream& _rawData);
```

Comment:
"fetchData" isn't the most descriptive, maybe something like "stringFromURL" or "streamFromURL"